python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -163,7 +163,7 @@ def num_nodes(self) -> Union[int, Dict[str, int]]:
         # Heterogenous
         else:
             num_nodes_per_type = {
-                _type: offset[_idx + 1] - offset[_idx]
+                _type: int(offset[_idx + 1] - offset[_idx])
```

Comment:
```suggestion
                _type: (offset[_idx + 1] - offset[_idx]).item()
```
Use `item` operation may be more formal?